integration_tests/snapshots/return_values/http-errors_node10_api-gateway-get.json

```diff
@@ -0,0 +1,39 @@
+{
+    "errorType": "Error",
+    "errorMessage": "getaddrinfo ENOTFOUND httpstat.us/400 httpstat.us/400:443",
+    "trace": [
+        "Error: getaddrinfo ENOTFOUND httpstat.us/400 httpstat.us/400:443",
+        "    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:56:26)"
+    ]
+}
+ 
+  Error --------------------------------------------------
+ 
+  Error: Invoked function failed
```

Comment:
Is there a way we can cut out all this serverless framework boilerplate from the return value?

Reply:
Good call, I've fixed these too with the correct `http` options for the function.